index.html

```diff
@@ -76,7 +82,7 @@ <h1>Tonic</h1>
                         <p>javaScript</p>
                     </div>
                     <div>
-                        <button class="btn"><a class="btn-text" href="#">See Project</a></button>
+                        <button class="btn " id="card-1" onclick="spreadB('card-1')"><a class="btn-text" href="#header-el">See Project</a></button>
                     </div>
                 </div>
             </div>
```

Comment:
- [ ] This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript. On page load, your javascript file should populate the projects list. Both the projects list populate and popup should render from your js file. Kindly remove all the html codes responsible for projects list and render them dynamically from your js file on page load. This is in line with the project requirements. Kindly take a look

![your-page](https://user-images.githubusercontent.com/26597432/203984569-1f0407bf-e16d-4f8b-b432-b6a109708173.png)

Reply:
- [ ] Kindly make to work on this requirement. On page load, your javascript file should populate the projects list.